IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinUtilBase.m

```diff
@@ -271,12 +271,40 @@ + (MSIDWPJKeyPairWithCert *)getWPJKeysWithTenantId:(__unused NSString *)tenantId
             return legacyKeys;
         }
     }
-     
+    
```

Comment:
Cherry picked from https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/1220